### PR TITLE
[UIPO-38] brands and shipping info

### PR DESCRIPTION
**Pull request by olenakachmar**

### CODE REVIEW

File: src/components/Brands/Brands.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  .container {
+    justify-content: center;
+  }
+}
+
+@media screen and (max-width: 768px) {
+  .brand {
+    &__list {
+      &-item {
```
Comment (Hollister009) [inline src/components/Brands/Brands.scss]: can be simplified on 1 level

File: src/components/Brands/Brands.spec.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+
+import { mount } from 'enzyme';
+import Brands from './Brands';
+
+const mockData = [
```
Comment (Hollister009) [inline src/components/Brands/Brands.spec.js]: put this mockData for reusability into a separate file

File: src/components/ShippingInfo/ShippingInfo.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import cx from 'classnames';
+
+import config from '../../../config';
+
+import './ShippingInfo.scss';
+
+export const CN = 'shippingInfo';
+const { shippingData } = config;
```
Comment (Hollister009) [inline src/components/ShippingInfo/ShippingInfo.js]: hmm, this import should be on line 4

File: src/components/ShippingInfo/ShippingInfo.spec.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  });
+
+  test('renders', () => {
+    wrapper = shallow(<ShippingInfo />);
+    expect(wrapper.exists()).toBe(true);
+  });
+
+  test('should match snapshot', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
```
Comment (Hollister009) [inline src/components/ShippingInfo/ShippingInfo.spec.js]: Better just test if it matches snapshot. There's no need to test for render

**Review by Hollister009 — COMMENTED** [review]
Overall, nice job!

File: src/components/Brands/Brands.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  componentDidMount() {
+    this.downloadBrands();
+  }
+
+  render() {
+    const { brandsList } = this.state;
+    const brands = brandsList.map((brand) => (
+      <li key={brand.id} className={`col-2 ${CN}__list-item`}>
+        <img src={brand.photo} className={`${CN}__list-photo`}/>
+      </li>
```
Comment (Nazar-bnm) [inline src/components/Brands/Brands.js]: move it to class method, for example renderBrendes() {}

File: src/components/ShippingInfo/ShippingInfo.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+const ShippingInfo = () => {
+  const shippingInfo = shippingData.map((elem) => (
+    <li key={elem.number} className={`${CN}__list-item`}>
+      <span className={`${CN}__item-number`}>0{elem.number}.</span>
+      <div className={`${CN}__item-text__wrapper`}>
+        <h5 className={`${CN}__item-text__title`}>{elem.title}</h5>
+        <h6 className={`${CN}__item-text__description`}>{elem.description}</h6>
+      </div>
+    </li>
```
Comment (Nazar-bnm) [inline src/components/ShippingInfo/ShippingInfo.js]: add object destruction for elem

**Review by Nazar-bnm — CHANGES_REQUESTED** [review]

**Review by Nazar-bnm — APPROVED** [review]

**Pull request merged**